src/Files.App/Converters/ConflictResolveOptionToIndexConverter.cs

```diff
@@ -3,9 +3,9 @@
 
 using Microsoft.UI.Xaml.Data;
 
-namespace Files.App.ValueConverters.EnumConverters
+namespace Files.App.Converters
 {
-	internal sealed class ConflictResolveOptionToIndexConverter : IValueConverter
+	internal class ConflictResolveOptionToIndexConverter : IValueConverter
```

Comment:
Why are you removing `sealed`? No class inherits from this one. In fact, I think we should apply that to other converters as well

Reply:
That’s should be accorded with guidline

Reply:
We may in turn want to make types sealed as many as possible as it can reduce virtual calls.

Reply:
Added to all of them.